doc/libcfu.texi

```diff
@@ -509,6 +509,11 @@ cfulist_map().  The return value is used to build a new list.
  Pop a value from the end of the list (removing it from the list).
 @end deftypefun
 
+@deftypefun void cfulist_delete_data (cfulist_t * @var{list}, void * @var{data})
```

Comment:
{void}

Reply:
Oh, you were referring to the first void, sorry, pushed now a fix! :)